docs/en/observability/cloud-monitoring/aws/monitor-amazon-intro.asciidoc

```diff
@@ -18,20 +20,33 @@ deep insights into the performance of your applications, services, and
 infrastructure components.
 
 Learn how to use the Elastic Observability solution to observe and monitor a
-broad range of AWS resources and applications. If the service or resource you
+broad range of AWS resources and applications. 
+
+- <<monitor-aws-elastic-agent,Monitor {aws} with {agent}>>
+- <<monitor-aws,Monitor {aws} with {beats}>>
+- <<monitor-aws-firehose,Monitor {aws} with Amazon Data Firehose>>
+- <<monitor-aws-esf,Monitor {aws} with Elastic Serverless Forwarder>>
+
+If the service or resource you
 want to monitor isn't shown here, go to the {integrations-docs}[Elastic
 Integrations docs] for a full list of supported AWS integrations.
 
```

Comment:
I think this still needs some work. A few thoughts:

* There's no link to the ingestion options page. We should either link to that page and encourage users to visit it, or...move that content to this page? It could actually fit well on this page instead of being it's own page.
* This page is about the different ways to ingest data, not about what's being monitored. The "If the service or resource you want to m...." link doesn't make sense to me in this context.

Reply:
Reworked the section monitoring intro page. The Ingestion options page has more visibility if we keep it separate. It is useful to keep the link to the Integrations AWS doc page, but I can rephrase it.